static/js/Story.jsx

```diff
@@ -157,7 +157,7 @@ const TopicTextPassage = ({text, topic, bodyTextIsLink=false, langPref, displayD
         return null;
     }
     const langKey = Sefaria.interfaceLang === 'english' ? 'en' : 'he';
-    const isCurated = text.descriptions && text.descriptions[langKey] && text.descriptions[langKey].title;
+    const isCurated = text.descriptions?.[langKey]?.title ?? false;
```

Comment:
I never knew`??` existed! It's not really necessary here because the right side is just a boolean but pretty neat.